src/chi/components/sidebar/sidenav.scss

```diff
@@ -376,6 +398,134 @@ $sidenav-lg-width: 12.5rem;
       }
     }
   }
+
+  &.-global-nav {
+    &.-inverse {
+      .m-sidenav__content {
+        background-color: set-color(grey, 100);
+        border-right: 0;
+
+        nav {
+          ul {
+            &.m-sidenav__list {
+              li {
+                border-bottom: 0;
+
+                &.-active {
+                  a {
+                    background-color: rgba(set-color(mint, 40), 0.08);
+
+                    &.unselected {
+                      background-color: transparent;
+                      box-shadow: none;
+                    }
+                  }
+                }
+
+                &:not(.-active) {
+                  a.-active {
+                    background-color: set-color(grey, 90);
+                    box-shadow: inset 0.125rem 0 0 set-color(white);
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+
+      .m-sidenav__drawers {
+        .m-drawer {
+          background-color: set-color(grey, 90);
+
+          &.-active {
+            .m-drawer__header {
+              align-items: center;
+              background-color: rgba(set-color(grey, 100), 0.5);
+              color: set-color(grey, 40);
+              display: flex;
+              font-size: 16px;
```

Comment:
@janibolkvadze please remove this property and add it to .a-drawer__title and convert it to rem